samples/core/src/main/java/io/oasp/gastronomy/restaurant/general/dataaccess/api/BinaryObjectEntity.java

```diff
@@ -1,13 +1,13 @@
 package io.oasp.gastronomy.restaurant.general.dataaccess.api;
 
-import io.oasp.gastronomy.restaurant.general.common.api.BinaryObject;
-
 import java.sql.Blob;
 
 import javax.persistence.Entity;
 import javax.persistence.Lob;
 import javax.persistence.Table;
 
+import io.oasp.gastronomy.restaurant.general.common.api.BinaryObject;
```

Comment:
@anmaso , Not sure how this order got changed . Reverted back the order and checked in the file.